src/applications/gqrx/mainwindow.h

```diff
@@ -121,6 +121,8 @@ public slots:
     // dummy widget to enforce linking to QtSvg
     QSvgWidget      *qsvg_dummy;
 
+    QByteArray stateBeforeMinimize;
```

Comment:
The name should reflect whos state we are talking about here. I'm not sure why "minimize"?